api/validators/MerchStoreRequests.ts

```diff
@@ -109,6 +130,9 @@ export class MerchItem implements IMerchItem {
   @Allow()
   hidden?: boolean;
 
+  @Allow()
+  hasVariantsEnabled?: boolean;
```

Comment:
Instead of setting the field as optional, can we specify a default through class-validator? If not, let's set a default in the controller.

Reply:
Looked into class-validator and it looks like the convention's to keep it read-only for validation. It should be good to just have a default set in the controller and specify this behavior in the API docs.